app/decorators/profile_decorator.rb

```diff
@@ -0,0 +1,14 @@
+class ProfileDecorator < ApplicationDecorator
+  DEV_HEADER_FIELDS = %w[employment_title employer_name].freeze
+
+  def ui_attributes_for(area:)
+    # Return a Hash of the profile fields that should be rendered for a given
+    # display area, e.g. :left_sidebar
+    names = ProfileField.public_send(area).pluck(:attribute_name)
+    # Temporary workaround: DEV specific header fields are hardcoded in the view
+    if SiteConfig.dev_to?
+      names -= DEV_HEADER_FIELDS
```

Comment:
Eventually, if we build the features to allow DEV's multi-field ProfileField (employer) we can remove this line.

Reply:
There are currently no plans for multi-field profile fields and unless we have another potential use case for them outside this I'd be very reluctant to change that, as we are bound to get the abstraction wrong.

Reply:
That's a fair point, maybe the goal should be to inch DEV closer to the vanilla Forem setup than to build a feature to keep this in DEV. Does the way I've written this make sense for now?

Reply:
> That's a fair point, maybe the goal should be to inch DEV closer to the vanilla Forem setup than to build a feature to keep this in DEV.

This is generally the direction we're taking right now. 😊

> Does the way I've written this make sense for **now?**

Absolutely 👍